client/dashboard/sites/settings-site-visibility/index.tsx

```diff
@@ -91,6 +92,7 @@ export default function SiteVisibilitySettings( { siteSlug }: { siteSlug: string
 				/>
 			}
 		>
+			<TrialUpsellNotice site={ site } />
```

Comment:
This line feels like it belongs in `launch-form.tsx`. It's the component that has the logic for disabling the launch button; it feels like it should also be responsible for rendering the warning for why that is.

Reply:
Updated in dfb04a7.